environment/src/main/java/com/toggl/environment/services/analytics/Event.kt

```diff
@@ -55,5 +55,8 @@ class Event private constructor(val name: String, val parameters: Map<String, St
                 "DistinctWorkspaceCount" to workspaceCount.toString()
             ) + providerCounts
         )
+
+        fun groupTimeEntriesSettingsChanged(state: Boolean) =
+            Event("GroupTimeEntriesSettingsChanged", mapOf("State" to if (state) "True" else "False"))
```

Comment:
I'm totally not in Aurora frame of mind yet, just wanted to offer a tiny suggestion

```suggestion
            Event("GroupTimeEntriesSettingsChanged", mapOf("State" to state.toString().capitalize()))
```

Not sure if it actually reads better or not 😄

Reply:
I'm not sure either but it's shorter so I guess I'll take it.